content/docs/overview/tracers.md

```diff
@@ -1,3 +1,51 @@
+
+### Setting up Tracer
+
+The first step in instrumenting your application with OT specification is setting up the tracer configuration and registering the tracer with GlobalTracer using the `register` method. This step basically indicates the tracer’s endpoint and the service that will send data to the server. The GlobalTracer forwards all the methods to the tracer that has been configured.
+The `register()` method can only be called once by the developer in every application during the initialization of a tracer. In case no Global Tracer is registered, the default `NoopTracer` will be used.
+
+### Starting a new Trace
+Finally, a new trace can be started by creating a new span. A span must specify the operation name. Operation names are used for providing semantic meaning to the trace. It is important to recognize whether we need to start a new trace or connect with an already-active trace. If there's an active trace, you'll start a span with the active span as its parent. Otherwise, the span you start will have no parent.OpenTracing supports two kinds of direct causal relationships:
+
+1. **ChildOf**: The parent span is dependent on the child span for its execution.
+2. **FollowsFrom**: This relationship is used to model asynchronous executions. The parent span is fully independent of the child span.
+
+
+### Accessing the Active Span
+The GlobalTracer can be used to access the currently ActiveSpan and log events. Further, ActiveSpans can be accessed through a scopeManager in some languages. Refer to the specific language guide for more implementation details.
+
+### Propagating a Trace with Inject/Extract
+In order to trace across process boundaries in distributed systems, services need to be able to continue the trace injected by the client that sent each request. OpenTracing allows this to happen by providing inject and extract methods that encode a span's context into a carrier.
+The inject method allows for the SpanContext to be passed on to a carrier. For example, passing the trace information into the client's request so that the server you send it to can continue the trace. The extract method does the exact opposite. It extract the SpanContext from the carrier. For example, if there was an active request on the client side, the developer must extract the span context using the `io.opentracing.Tracer.extract` method.
+
+## Tracing Systems
+
+The following tracing backends have OpenTracing-compatible tracing libraries:
+
+Tracing System | Supported Languages
+------------ | -------------
+CNCF Jaeger | [Java](https://github.com/jaegertracing/jaeger-client-java), [Go](https://github.com/jaegertracing/jaeger-client-go), [Python](https://github.com/jaegertracing/jaeger-client-python), [Node.js](https://github.com/jaegertracing/jaeger-client-node), [C++](https://github.com/jaegertracing/cpp-client), [C#](https://github.com/jaegertracing/jaeger-client-csharp)
```

Comment:
Why is Jaeger the first item? Is this table sorted in some way? It would probably be better to sort it alphabetically.